content/accessibility/truncation.mdx

```diff
@@ -0,0 +1,95 @@
+---
+title: Truncation
+---
+
+## Overview
+
+Truncation means shortening a text when it reaches a larger length than required, usually shorten with ellipses.
+
+There are several parts of our platform where we will have some text, which can probably contain focusable elements like links,
+and we won't have enough space to show it completely, so we wan't to truncate it. How do we truncate it in the most accessible way?
+
+The answer is unfortunately that we need to be a lot more explicit about how that truncation happens.
+We would need to identify a couple of use cases to understand the way that truncation can be done.
+
+## Implicit vs Explicit truncation
+
+### Implicit truncation
+
+Implicit truncation means that a certain text is truncated and the user has to take an action to make the text appear in full, although that action is not an explicit action using a control in the interface, meaning that the user can only expand by hovering or focusing it.
+
+### Explicit truncation
+
+Explicit truncation means that a certain text is truncated and it includes an actual control in the interface that can be clicked on or pressed with the keyboard, letting the user expand what is displayed.
+
+## General Rules
+
+### Focusable elements
+
+As a general rule we should not use implicit truncation text that can contain focusable elements.
+
+Some of the potential issues are:
+- the focusable elements could be completely hidden and they will randomly appear when the user navigates into them.
+- the focus seems to disappear.
+- focusable elements can be truncated in the middle.
+- "unsecure" links can be focused via keyboard whith no visual focus.
+
+### Security
+
+We don't want to ever truncate something that causes any potential security concerns.
+For example, truncating a URL is a security bad practice, because the user doesn't have insight into where the link might be going.
+
+### Untrue information
+
+In some situations we don't control in which part of the text the truncation is happening, so we wan't to avoid to truncate a text if that could be considered as 
```

Comment:
```suggestion
In some situations we don't control in which part of the text the truncation is happening, so we want to avoid truncated text if that could be considered as 
```